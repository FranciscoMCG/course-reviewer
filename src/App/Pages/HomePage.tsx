import React from "react";
import { Link } from "react-router-dom";

const HomePage = () => (
  <>
    <h1>Course reviews</h1>
    <div className="col-md-12">
      <div className="button-wrapper btn-container-center">
        <Link to="/reviews" className="link">
          See reviews
        </Link>
        <Link to="/reviews/create-review" className="link">
          Create a review
        </Link>
      </div>
    </div>
  </>
);

export default HomePage;
