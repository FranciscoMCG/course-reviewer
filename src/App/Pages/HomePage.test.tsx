import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import HomePage from "./HomePage";

describe("<HomePage/ >", () => {
  const wrapper = shallow(<HomePage />);
  it("should contain 1 h1 element", () => {
    expect(wrapper.find("h1").length).toBe(1);
  });
  it("should render component", () => {
    const wrapper = shallow(<HomePage />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
