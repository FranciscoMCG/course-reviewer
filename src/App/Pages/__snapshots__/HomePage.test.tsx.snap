// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage/ > should render component 1`] = `
<Fragment>
  <h1>
    Course reviews
  </h1>
  <div
    className="col-md-12"
  >
    <div
      className="button-wrapper btn-container-center"
    >
      <Link
        className="link"
        to="/reviews"
      >
        See reviews
      </Link>
      <Link
        className="link"
        to="/reviews/create-review"
      >
        Create a review
      </Link>
    </div>
  </div>
</Fragment>
`;
